Modules/_functoolsmodule.c

```diff
@@ -46,9 +46,10 @@ typedef struct {
     PyObject *fn;
     PyObject *args;
     PyObject *kw;
-    PyObject *dict;        /* __dict__ */
-    PyObject *weakreflist; /* List of weak references */
+    PyObject *dict;         /* __dict__ */
+    PyObject *weakreflist;  /* List of weak references */
     vectorcallfunc vectorcall;
+    Py_ssize_t can_vcall;   /* Cache whether function allows vector call */
```

Comment:
It might be better to not change the comment allignment here.

```suggestion
    PyObject *dict;        /* __dict__ */
    PyObject *weakreflist; /* List of weak references */
    vectorcallfunc vectorcall;
    Py_ssize_t can_vcall;  /* Cache whether function allows vector call */
```

Reply:
Are there any specific rules here? In python I personally have a habit of keeping at least 2 spaces. What is the general practice when writing extensions?

Reply:
In the examples a single space is used: https://peps.python.org/pep-0007. But you should mainly try to avoid unnecessary stylistic changes.

Reply:
Noted. I think it is cleaner like this now opposed to adding an extra commit for 2 spaces, but will correct it if there is a chance or big need for that.

Reply:
We generally try to not make unnecessary changes to keep git blame clean. All commits are squashed anyway, so it doesn't matter how many commits you make.